pallets/qa-tools/src/lib.rs

```diff
@@ -31,6 +31,8 @@
 #![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
It would be nice to have a comment with pallet description. It might be a some basic info from the https://github.com/sora-xor/sora2-network/issues/749.